include/RAJA/exec-openmp/scan_openmp.hxx

```diff
@@ -82,6 +82,7 @@ void inclusive_scan_inplace(const omp_parallel_for_exec&,
                             ValueT v)
 {
   using Value = typename std::decay<decltype(*std::declval<Iter&>())>::type;
+  // using Value = typename std::iterator_traits <Iter>::value_type;
```

Comment:
I'd agree with that assessment.  Is there a chance that it's meant to be an `int*` and the pointer qualifier is getting dropped?